types/packages/caver-core/src/index.d.ts

```diff
@@ -287,3 +307,22 @@ export interface EncryptionOptions {
     cipher?: any
     uuid?: any
 }
+
+export interface FeeHistoryResult {
+    oldestBlock: number
+    baseFeePerGas: string[]
+    reward: string[][]
+    gasUsedRatio: number[]
+}
+
+export interface AccessListResult {
+    accessList: AccessList
+    gasUsed: string
+}
```

Comment:
@jimni1222 

The `error` should be added because [AccessListResult](https://github.com/klaytn/klaytn/blob/e7a8884b3e97920da3a67cf84a683a0551c1e927/api/api_public_blockchain.go#L420-L427) contains error field.
This field is used when there was an error occurred during EMV execution.